src/main/java/hudson/plugins/git/GitChangeSet.java

```diff
@@ -356,7 +357,21 @@ String getParentCommit() {
      * @param createAccountBasedOnEmail true if create new user based on committer's email.
      * @return {@link User}
      */
+    @Obsolete
```

Comment:
I don't understand the usage of the Apache http Obsolete annotation for this case.  The [Obsolete annotation javadoc](https://hc.apache.org/httpcomponents-core-ga/httpcore/apidocs/org/apache/http/annotation/Obsolete.html) says:

> The field or method to which this annotation is applied is marked as implementing requirements of the HTTP protocol or a related protocol that are now obsolete.

This method is unrelated to an HTTP protocol implementation.  Could you instead mark it as Deprecated both with a Deprecated annotation and with a javadoc entry that it is deprecated?  Other methods in the plugin which have been superseded are marked with the Deprecated annotation.

Could you also include a  `see` reference in the javadoc comment for the replacement method?

Reply:
Sorry, my .NET background kicked in, and the IDE auto import did the rest.